hz.ps1

```diff
@@ -2562,6 +2590,30 @@ function hz-getfwks-json {
     ConvertTo-Json -InputObject $frameworks -Compress
 }
 
+## Copies files from given path to project folder with respect to source hierarhcy.
+function hz-copy-source (){
+
+    $source = $options.source
+
+    if(-not (test-path -path $source)){
+        Die "$($source) is not exist."
+    }
+
+    $currentPath = get-location
```

Comment:
not sure I like this. the script *could* potentially run from anywhere. we have a range of defined directories (see top of `hz.ps1`) including the solution's root. think it would be more reliable to reference one of these instead of "the current location"